frontend/src/app/utils/__snapshots__/srGraphQLErrorMessage.test.tsx.snap

```diff
@@ -26,13 +28,15 @@ Object {
       <span>
          
         <a
-          href="mailto:support@simplereport.gov"
+          href="/contact-us"
+          target="_blank"
```

Comment:
every time an anchor has a target set to _blank it needs to have this attribute as well `rel="noopener noreferrer"`